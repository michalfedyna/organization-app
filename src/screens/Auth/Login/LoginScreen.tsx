import React from 'react';

import Layout from '@components/Layout';

const LoginScreen = () => {
  return <Layout.Container />;
};

export default LoginScreen;
