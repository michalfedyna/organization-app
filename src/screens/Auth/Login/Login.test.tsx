import React from 'react';
import {test, expect} from '@jest/globals';
import renderer from 'react-test-renderer';

import LoginScreen from '@screens/Auth/Login/LoginScreen';

test('LoginScreen renders correctly', () => {
  const tree = renderer.create(<LoginScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
