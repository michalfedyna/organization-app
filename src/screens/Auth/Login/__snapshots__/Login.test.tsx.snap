// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreen renders correctly 1`] = `
<View
  style={
    {
      "flexDirection": "column",
    }
  }
/>
`;
